test/e2e/network/service.go

```diff
@@ -303,11 +303,11 @@ func StopServeHostnameService(clientset clientset.Interface, ns, name string) er
 }
 
 // verifyServeHostnameServiceUp wgets the given serviceIP:servicePort from the
-// given host and from within a pod. The host is expected to be an SSH-able node
-// in the cluster. Each pod in the service is expected to echo its name. These
-// names are compared with the given expectedPods list after a sort | uniq.
+// given host or from within a pod(if there are no host satisfied, we will only verify from pod).
+// The host is expected to be an SSH-able node in the cluster. Each pod in the service is expected to echo its name.
+// These names are compared with the given expectedPods list after a sort | uniq.
```

Comment:
IMO, we still need check hoth host network and contianer network, our target is to use `framework.RunHostCmd` replace `e2essh.SSH`.

Reply:
You are right. I should think it more carefully. 👍